packages/friends/yml/en/francis-howgill.yml

```diff
@@ -2,6 +2,24 @@ id: 311c83ee-2bb9-40fb-9ff5-6e34ea088ac7
 name: Francis Howgill
 slug: francis-howgill
 gender: male
+born: 1618
+died: 1669
+quotes:
+  - source: Francis Howgill
```

Comment:
I love this quote. I wonder if we could make the source `Francis Howgill (Applyby Jail)` or whatever it was named, to underscore that he was dying in prison.  Would that work you think?